src/AnalyticsValues.php

```diff
@@ -45,13 +45,10 @@ public static function from_smartsheets(
 		$approved_column     = $global_smartsheet->get_column_index( 'Standard Approved' );
 		$tracking_id_column  = $global_smartsheet->get_column_index( 'Tracking ID' );
 
-		$global_projects = $global_smartsheet->filter_by_column( $approved_column, true )->sort_on_column(
-			$project_name_column
-		)->export_columns(
-			[
-				$project_name_column => 'global_project_name',
-				$tracking_id_column  => 'tracking_id',
-			]
+		$global_projects = $global_smartsheet
+			->filter_by_column( $approved_column, true, true )
```

Comment:
The API response of Smartsheet contained a boolean, while for Docs it's just the cell content.

For now the implementation requires to use a value that PHP type casts to either true or false.

I might change this to do a different filter depending on the source sheet.